xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-test/xwiki-platform-ckeditor-test-docker/pom.xml

```diff
@@ -58,6 +58,20 @@
       <version>${project.version}</version>
       <type>xar</type>
     </dependency>
+    <dependency>
+      <!-- We need this to test the mentions Quick Action. -->
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-mentions-ui</artifactId>
+      <version>${project.version}</version>
+      <type>xar</type>
+    </dependency>
+    <dependency>
+      <!-- We need this to test the code snippet Quick Action. -->
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-rendering-macros</artifactId>
+      <version>${project.version}</version>
+      <type>pom</type>
```

Comment:
Depending on the POM looks strange. Don't you need to depend on ``xwiki-platform-rendering-macro-code`` instead?

Reply:
Yes. The point of depending on a pom is to get the dependencies of a module without the module itself, but it does not seem to be something needed here.

That being said, it seems strange to need the code macro for a test. If you added it because it's used by a XAR you are testing then it should be added as dependency of that XAR instead.